pytorch/training/docker/1.4.0/py3/Dockerfile.cpu

```diff
@@ -96,13 +98,18 @@ RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-lat
 RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
  && pip install --no-cache-dir -U \
+    # install PyYAML>=5.4,<5.5 to avoid conflict with latest awscli
+    "PyYAML>=5.4,<5.5" \
     "awscli<2" \
+    "cryptography>3.2" \
+    ruamel-yaml \
     fastai==1.0.59 \
     scipy==1.2.2 \
     smdebug==${SMDEBUG_VERSION} \
-    "sagemaker<2" \
+    "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
     "sagemaker-pytorch-training<2" \
+    sagemaker-training \
```

Comment:
`sagemaker-training` is already a part of the requirements of `sagemaker-pytorch-training` and doesn't need to be installed again.

Reply:
We still see the error in the test ` ModuleNotFoundError: No module named 'sagemaker_training'`

Checking with the PT team, they recommended to install this explicitly.